### PR TITLE
Correcting filename example.env to example.env.yml

### PR DESCRIPTION
**Pull request by kidsil**

### CODE REVIEW

**Comment by kidsil** [comment]
Unless I'm mistaken and it needs to be called _.env.yml_ ? or _env.yml_
At any case it needs to be corrected..

**Comment by laardee** [comment]
@kidsil thanks, you are correct 👍. 

The renamed should be called _env.yml_.

I did this environmental variable change yesterday and this is the result what you get when you make a PR at the middle of the night and review it yourself 😁 ...

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ If you are using Serverless framework v.0.5, see branch https://github.com/laard
 Installation will create one DynamoDB table for OAuth state and refresh tokens.
 
 1. Run `serverless install --url https://github.com/laardee/serverless-authentication-boilerplate`, clone or download the repository
-2. Rename _example.env_ in _authentication_ to _.env_ and set [environmental variables](#env-vars).
+2. Rename _example.env.yml_ in _authentication_ to _.env_ and set [environmental variables](#env-vars).
```
Comment (laardee) [inline README.md]: _.env_ should be  _env.yml_

Comment (kidsil) [inline README.md]: done

**Pull request merged**